content/id/docs/concepts/containers/images.md

```diff
@@ -0,0 +1,370 @@
+---
+title: Image
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Kamu membuat Docker _image_ dan mengunduhnya ke sebuah registri sebelum digunakan di dalam Kubernetes Pod.
+
+Properti `image` dari sebuah Container mendukung sintaksis yang sama seperti perintah `docker`, termasuk registri privat dan _tag_.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Memperbarui Image
+
+Kebijakan _pull default_ adalah `IfNotPresent` yang membuat Kubelet tidak
+lagi mengunduh (_pull_) sebuah image jika sudah ada terlebih dahulu. Jika kamu ingin agar
+selalu diunduh, kamu bisa melakukan salah satu dari berikut:
+
+- mengatur `imagePullPolicy` dari Container menjadi `Always`.
+- buang `imagePullPolicy` dan gunakan `:latest` _tag_ untuk _image_ yang digunakan.
+- buang `imagePullPolicy` dan juga _tag_ untuk _image_.
+- aktifkan [AlwaysPullImages](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages) _admission controller_.
+
+Harap diingat kamu sebaiknya hindari penggunaan _tag_ `:latest`, lihat [panduan konfigurasi](/docs/concepts/configuration/overview/#container-images) untuk informasi lebih lanjut.
+
+## Membuat Image Multi-arsitektur dengan Manifest
+
+Docker CLI saat ini mendukung perintah `docker manifest` dengan anak perintah `create`, `annotate`, dan `push`. Perintah-perintah ini dapat digunakan
+untuk membuat (_build_) dan mengunggah (_push_) _manifest_. Kamu dapat menggunakan perintah `docker manifest inspect` untuk membaca _manifest_.
```

Comment:
```suggestion
untuk membuat (_build_) dan mengunggah (_push_) manifes. Kamu dapat menggunakan perintah `docker manifest inspect` untuk membaca manifes.
```

there is _manifes_ in kbbi
https://kbbi.kemdikbud.go.id/entri/manifes

Reply:
ah I just learned this, thanks for pointing out, this means we can also introduce `manifes` to other translated pages

Reply:
fixed in https://github.com/kubernetes/website/pull/19784/commits/3fb85d866101c08ce75ea0265beac4929e73f29c